web/html/src/flow-typed/suma.js

```diff
@@ -0,0 +1,2 @@
+declare function t(msg: string): string;
```

Comment:
This files is a good addition and makes a lot of sense to me. :+1: 
Just one thing: `suma.js` is not a good name choice here, we should at least name it `susemanager.js`, but then, we would have to rename it to `uyuni.js` for this proper PR/git-repo. I would better name it `custom-functions.js`, `additionals.js`, or something like this, just to be abstract enough from the context.

Reply:
Changes it to "globals.js"